packages/block-library/src/latest-posts/edit.native.js

```diff
@@ -63,9 +62,7 @@ class LatestPostsEdit extends Component {
 			.then( ( categoriesList ) => {
 				if ( this.isStillMounted ) {
 					this.setState( {
-						categoriesList: isEmpty( categoriesList )
-							? []
-							: categoriesList,
+						categoriesList,
```

Comment:
Could this be something else than an empty array?

Reply:
Nope, it will always be an array, even if in a very rare scenario where there are no category terms in the database, it would be an empty one:

https://github.com/WordPress/wordpress-develop/blob/e50c65caf4999b11eb396ceaf6649b9818fab47f/src/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php#L340

Reply:
Thank you @tyxla!